contracts/0.4.24/Lido.sol

```diff
@@ -455,29 +444,32 @@ contract Lido is StETH, AragonApp {
         // EL values
         uint256 _withdrawalVaultBalance,
         // decision
-        uint256 _withdrawalsReserveAmount,
+        uint256 _newBufferedEtherReserveAmount,
         uint256[] _requestIdToFinalizeUpTo,
         uint256[] _finalizationShareRates
     ) external {
         require(msg.sender == getOracle(), "APP_AUTH_FAILED");
         _whenNotStopped();
 
-        // update withdrawals reserve
-        WITHDRAWAL_RESERVE_POSITION.setStorageUint256(_withdrawalsReserveAmount);
-
         uint256 preBeaconBalance = BEACON_BALANCE_POSITION.getStorageUint256();
 
+        // updating saved stats from beacon balance checking its sanity
```

Comment:
"updating" --> "update" to make uniform with other nearby comments

but sorry, yeah, this is not of importance